CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 
+# 0.12.1 (Aug 30, 2019)
+
+-   **[Fix]**: Fix an error when updating streamId for subscriber component (merged from [PR326](https://github.com/opentok/opentok-react-native/pull/326)). Adheres to: [#315](https://github.com/opentok/opentok-react-native/issues/315), [#324](https://github.com/opentok/opentok-react-native/issues/324)
+-   **[Fix]**: Use Context API for passing props down the component tree (merged from [PR333](https://github.com/opentok/opentok-react-native/pull/333)). Adheres to: [#329](https://github.com/opentok/opentok-react-native/issues/329), [#335](https://github.com/opentok/opentok-react-native/issues/335)
+-   **[Fix]**: Fix `Attempted to register RCTBridgeModule class OTSessionManager` issue after opening/reloading app (merged from [PR336](https://github.com/opentok/opentok-react-native/pull/336)).
+-   **[Fix]**: Fix reload in iOS (merged from [PR339](https://github.com/opentok/opentok-react-native/pull/339)).
+-   **[Fix]**: Fix security vulnerabilities on deps (merged from [PR339](https://github.com/opentok/opentok-react-native/pull/340)).
```

Comment:
How about `dependencies` instead of `deps`? 
Besides that, LGTM!

Reply:
done!